images/router/haproxy/Dockerfile

```diff
@@ -5,13 +5,12 @@
 #
 FROM openshift/origin-cli
 
-COPY rhel-server-ose.repo /etc/yum.repos.d
-# If this is a non-CI/RHEL build, the repo is not necessary
-RUN test -e /etc/centos-release || rm /etc/yum.repos.d/rhel-server-ose.repo
+RUN yum install -y https://raw.githubusercontent.com/candita/haproxy-testing/main/haproxy18-1.8.28-1.el7.x86_64.rpm 
+RUN haproxy -vv
 
-RUN INSTALL_PKGS="haproxy18 rsyslog" && \
+RUN INSTALL_PKGS="rsyslog" && \
     yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
+    rpm -V haproxy18 $INSTALL_PKGS && \
```

Comment:
Drop the reference to `haproxy18` it should have been installed by the `yum install`. At the very least just install the one we want to test CI with.

Reply:
Or move the `RUN haxproxy -vv` to after all the package installs to verify what version we're testing with post RPM install.

Reply:
That works for `yum` but not for `rpm`.  I had to use the alias for `rpm -V`

Reply:
OK, I misread this a bit. Apologies. The times I have done this I avoid any reference to `haproxy18` as I install it directly from a known place and the `haproxy -vv` will confirm build+version info. This is fine. Apologies for the noise; it just looked different to the patterns I had ...

Reply:
You have a point, it's better to keep it as clean as possible for future reference, so I updated it.